proto/v1/project_service.proto

```diff
@@ -323,7 +323,7 @@ message Binding {
   // Specifies the principals requesting access for a Bytebase resource.
   // `members` can have the following values:
   //
-  // * `user:{emailid}`: An email address that represents a specific Bytebase
```

Comment:
Here we should still use user:{email} to be consistent with GCP IAM.

Reply:
https://cloud.google.com/iam/docs/reference/rest/v1/Policy#:~:text=An%20Identity%20and%20Access%20Management,(such%20as%20G%20Suite).